x-pack/plugins/cases/public/components/recent_cases/filters/index.test.tsx

```diff
@@ -50,13 +48,13 @@ describe('Severity form field', () => {
   it('selects the correct value when changed to reported by me', async () => {
     const result = appMockRender.render(<RecentCasesFilters {...props} />);
 
-    userEvent.click(result.getByTestId('recent-cases-filter'));
+    const recentCasesFilter = result.getByTestId('recent-cases-filter');
 
-    await waitForEuiPopoverOpen();
+    expect(recentCasesFilter).toBeInTheDocument();
 
-    expect(result.getByTestId('recent-cases-filter-myRecentlyReported')).toBeTruthy();
+    expect(result.getByText(caseFilterOptions[1].label)).toBeInTheDocument();
```

Comment:
nit: You can do `result.getByLabelText`

Reply:
It is not a label actually. It is just a text of select option.
```
`<option value="myRecentlyReported">
  Created by me
 </option>
`
```

Reply:
oh, I see. I was confused with `.lable`. All good 🙂.